extension/package.json

```diff
@@ -1351,7 +1351,7 @@
       {
         "view": "dvc.views.webviews",
         "contents": "[Show Experiments](command:dvc.showExperiments)\n[Show Plots](command:dvc.showPlots)\n[Show Experiments and Plots](command:dvc.showExperimentsAndPlots)",
-        "when": "dvc.commands.available && dvc.project.available"
+        "when": "true"
```

Comment:
[Q] Would it be easier to show the same view but with different underlying commands under different circumstances? Would that simplify the code elsewhere?

i.e 
```
      {
        "view": "dvc.views.webviews",
        "contents": "[Show Experiments](command:dvc.showExperiments)\n[Show Plots](command:dvc.showPlots)\n[Show Experiments and Plots](command:dvc.showExperimentsAndPlots)",
        "when": "dvc.commands.available && dvc.project.available"
},
      {
        "view": "dvc.views.webviews",
        "contents": "[Show Experiments](command:dvc.showEmptyWebview)\n[Show Plots](command:dvc.showEmptyWebview)\n[Show Experiments and Plots](command:dvc.showEmptyWebview)",
        "when": "!dvc.commands.available || !dvc.project.available"
}
```

Reply:
I think you're right. I'll try that. Thanks!